pkg/driver/nydus/nydus.go

```diff
@@ -210,17 +211,38 @@ func (d *Driver) Convert(ctx context.Context, provider accelcontent.Provider, so
 	if err != nil {
 		return nil, errors.Wrap(err, "get source image")
 	}
-	desc, err := d.convert(ctx, provider, *image, sourceRef)
+
+	//FIXME:here we just use the first part of sourceRef as source,we need to get remote cache name gracefully
+	//split sourceRef to get remote source
+	items := strings.Split(sourceRef, "/")
+	cacheNamed := items[0] + "/library/cache-manifest:latest-nydus"
```

Comment:
Maybe we can use `addSuffix` in `pkg/adapter/rule.go` directly, and make this configurable.